tests/integration/helpers/cluster.py

```diff
@@ -592,6 +608,18 @@ def start(self, destroy_dirs=True):
                 self.kafka_docker_id = self.get_instance_docker_id('kafka1')
                 self.wait_schema_registry_to_start(120)
 
+            if self.with_kerberized_kafka and self.base_kerberized_kafka_cmd:
+                env_var = {}
+                env_var['KERBERIZED_KAFKA_DIR'] = instance.path + '/'
+
+                # different docker_compose versions look for .env in different places
+                #   -- env-file too recent to rely on it
+                files_to_cleanup = []
+                files_to_cleanup.append(_create_env_file(self.base_dir, env_var, ".env"))
+                files_to_cleanup.append(_create_env_file(os.getcwd(), env_var, ".env"))
+                subprocess_check_call(self.base_kerberized_kafka_cmd + common_opts + ['--renew-anon-volumes'])
+                self.kerberized_kafka_docker_id = self.get_instance_docker_id('kerberized_kafka1')
+                remove_files(files_to_cleanup)
```

Comment:
That `env` manipulations don't look great.  normal env setup should just work (check for `MINIO_CERTS_DIR` for example)

Reply:
Yes, switching to variables.